frontend/src/plugins/router.js

```diff
@@ -45,13 +45,18 @@ function requireAuth(to, from, next) {
   const user = store.getters['user/get']
 
   if (user) {
-    next()
-  } else {
-    next({
-      path: '/user/login',
-      query: { redirect: to.fullPath }
-    })
+    if (user.timestamp > ((Date.now() - 1800000))) {
```

Comment:
@TDiercks Could you please extract the timeout value into a const and increase it to 24h as 30 Minutes always reminds me of my online banking applications that logs me out before I have even logged in 😄